application/dashboard/trello_service.py

```diff
@@ -90,12 +96,21 @@ def map_card(self, card):
             obj["stage"] = TRELLO_LISTS[card.idList]["stage"]
         return obj
 
-    def find_flag(self, card, flags):
+    @staticmethod
+    def _find_flag(card, flags):
         if card.labels:
             for flag in card.labels:
                 if flag.name in flags:
                     return flag.name
         return ""
 
+    @staticmethod
+    def _remove_internal_reference(card_name):
+        match = TrelloService.REMOVE_INTERNAL_REFERENCE_REGEX.match(card_name)
```

Comment:
would `return card_name.sub(TrelloService.REMOVE_INTERNAL_REFERENCE_REGEX, '', card_name)` be cleaner here? (If I've read the api docs correctly)

Reply:
Yep, although in this case the regex needs to be slightly different.

I've pushed up a new version that's neater - ta!